static/css/pad.css

```diff
@@ -57,6 +57,7 @@ textarea {
 
 /* Position the online count button a bit better */
 li[data-key=showusers] > a #online_count {
+    background-color: #000;
```

Comment:
Would it make sense to add the `online_count` span to the button itself to ensure that it just inherits the colors from the other buttons? That would automatically update it if we ever change our toolbar button colors.